### PR TITLE
Converting accountId and owner to lowercase for the comparison to avoid problems caused by inconsistent cases

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #1377

Changes proposed in this PR:

- Converting accountId and owner to lowercase for the comparison to avoid problems caused by inconsistent cases

### CODE REVIEW

File: src/components/Asset/Edit/index.tsx [hunk 1]
```diff
@@ -37,23 +37,23 @@ export default function Edit({ uri }: { uri: string }): ReactElement {
     }
   ].filter((tab) => tab !== undefined)
 
-  return asset && asset?.accessDetails && accountId === owner ? (
+  return !asset || !accountId || !owner || !asset?.accessDetails ? (
```
Comment (kremalicious) [inline src/components/Asset/Edit/index.tsx:40]: we should not add another set of crazy long clauses here, and continue to use the loader as the fallback only so we don't have to add every possible condition here. All those conditions should live on the next clause to define if edit should be shown or not.

**Comment by jamiehewitt15** [comment]
Actually, after a bit more testing it looks like this issue is caused by inconsistent capitalisation between the accountId / owner account

Account Id - 2022-05-02_13-01

**Review by claudiaHash — COMMENTED** [review]
After entering on an asset which I own page without a wallet connected, and then I connect my wallet, I got this error:

Screenshot from 2022-05-02 14-40-23

**Comment by jamiehewitt15** [comment]
> After entering on an asset which I own page without a wallet connected, and then I connect my wallet, I got this error:

@claudiaHash thanks for pointing that out. Should be working ok now. If you're on the edit page and disconnect your account you should see the "Edit action available only to asset owner" error message

**Review by claudiaHash — APPROVED** [review]
LGTM

File: src/components/Asset/Edit/index.tsx (hunk 1 above)
Comment (jamiehewitt15) [reply src/components/Asset/Edit/index.tsx]: I've removed this

File: src/components/Asset/Edit/index.tsx [hunk 2]
```diff
@@ -37,13 +37,17 @@ export default function Edit({ uri }: { uri: string }): ReactElement {
     }
   ].filter((tab) => tab !== undefined)
 
-  return asset && asset?.accessDetails && accountId === owner ? (
+  return asset &&
+    asset?.accessDetails &&
+    accountId?.toLowerCase() === owner?.toLowerCase() ? (
```
Comment (EnzoVezzaro) [inline src/components/Asset/Edit/index.tsx:42]: I guess is fine, but why are we getting this ID formatted differently?

**Review by EnzoVezzaro — APPROVED** [review]

File: src/components/Asset/Edit/index.tsx (hunk 2 above)
Comment (jamiehewitt15) [reply src/components/Asset/Edit/index.tsx:42]: I'm not really sure to be honest

**Pull request merged**